### DISCUSSION TITLE
Help needed: installation directory does not look right

### DISCUSSION THREAD

**Discussion by berrylizzard** [openvinotoolkit/openvino#16793] [General]
Hello! 

Some information about the systems I am working with:
* Ubuntu 22.04 (specifically [WSL](https://learn.microsoft.com/en-us/windows/wsl/about))
* Python 3.10.6

I ran the `pip install openvino-dev` command (as a superuser). After installation, `mo - h` resulted in the Model Optimizer help menu being displayed and I was able to `import openvino` in Python3. 

The issue comes up when I check the installation directory which I expect to be default (`/opt/intel/openvino-2022`), but my `/opt` directory is empty. Instead, if I rerun installation command I get `Requirement already satisfied: openvino-dev in /usr/local/lib/python3.10/dist-packages (2022.3.0)`. That directory looks like this:
```
/usr/local/lib/python3.10/dist-packages/openvino# tree -d -L 2
.
├── __pycache__
├── _offline_transformations
│   └── __pycache__
├── frontend
│   ├── __pycache__
│   ├── onnx
│   ├── paddle
│   └── tensorflow
├── helpers
│   └── __pycache__
├── inference_engine
│   └── __pycache__
├── libs
├── model_zoo
│   ├── __pycache__
│   ├── data
│   ├── download_engine
│   ├── internal_scripts
│   ├── models
│   └── quantize_engine
├── offline_transformations
│   └── __pycache__
├── preprocess
│   └── __pycache__
├── pyopenvino
│   └── __pycache__
├── runtime
│   ├── __pycache__
│   ├── op
│   ├── opset1
│   ├── opset10
│   ├── opset2
│   ├── opset3
│   ├── opset4
│   ├── opset5
│   ├── opset6
│   ├── opset7
│   ├── opset8
│   ├── opset9
│   ├── passes
│   └── utils
└── tools
    ├── __pycache__
    ├── accuracy_checker
    ├── benchmark
    ├── mo
    └── pot
```
I expect the installation directory to have a `deployment_tools` folder (to configure the Movidius MyriadX VPU as described in [configuration guide](https://docs.openvino.ai/2021.4/openvino_docs_install_guides_installing_openvino_linux_ivad_vpu.html)) yet as you can see, it's not there. The above directory tree is only 2 levels deep, but there are files and other folders inside the ones shown, let me know if that would be helpful to have and I'll provide it. 

Overall, my question is why didn't the installation happen in the `/opt/intel` directory by default and why am I missing folder(s) that are mentioned in the guide? 

Any pointers or ideas are welcome. Thanks!

P.S. As I was typing this question, I discovered that [Ubuntu 22.04 is not officially supported yet with OpenVINO toolkit, but can be installed with PyPI](https://www.intel.com/content/www/us/en/support/articles/000091952/software/development-software.html). If that is what causing my problems, is there any way to know whether Ubuntu 22.04 will be supported in the upcoming 2023.0 release? Is there an approximate timeline for 2023.0 release?

**Comment by ilya-lavrenov** [comment]
> /opt/intel/openvino-2022

pip (python package manager) never installs packages to such locations. It installs to current global / virtual environment.

> I expect the installation directory to have a deployment_tools

Since 2022.1 release, OpenVINO package structure has changes. 
Please, download OpenVINO archive and look inside. The package structure is much more clear now

>  why am I missing folder(s) that are mentioned in the guide?

Because you use wrong type of distribution. Use archive / debian package

> If that is what causing my problems, is there any way to know whether Ubuntu 22.04

While Ubuntu 22.04 is not explicitly mentioned in documentation, it's still supported. Binaries from Ubuntu 20.04 perfectly work on Ubuntu 22.04

**Reply by berrylizzard** [reply]
Thank you for reply, I'm glad the OS version is not an issue. 

Pardon my lack of knowledge, but what do you mean this?
> Because you use wrong type of distribution. Use archive / debian package
Building OpenVINO from source? Installing it with apt-get?

Originally I was following [the guide here](https://docs.openvino.ai/latest/openvino_docs_install_guides_install_dev_tools.html) which advises to use `pip`. 

Edit:
I've found an archive .tgz file [here](https://storage.openvinotoolkit.org/repositories/openvino/packages/2022.3/linux/) for Ubuntu 20.04 and de-archived it in my home directory into folder `openvino_toolkit_2022.3`. There, the structure is (with some output modification)
```
(openvino_env) xxx@xxx:~/openvino_toolkit_2022.3$ tree -L 2
.
├── docs
│   └── ...
├── install_dependencies
│   ├── install_NCS_udev_rules.sh
│   ├── install_NEO_OCL_driver.sh
│   └── install_openvino_dependencies.sh
├── python
│   ├── python3.10
│   ├── python3.7
│   ├── python3.8
│   └── python3.9
├── runtime
│   ├── cmake
│   ├── include
│   ├── lib
│   └── version.txt
├── samples
│   └── ...
├── setupvars.sh
└── tools
    ├── compile_tool
    ├── deployment_manager
    ├── requirements.txt
    ├── requirements_caffe.txt
    ├── requirements_kaldi.txt
    ├── requirements_mxnet.txt
    ├── requirements_onnx.txt
    ├── requirements_pytorch.txt
    ├── requirements_tensorflow.txt
    └── requirements_tensorflow2.txt

19 directories, 19 files
```
I did run the `setupvars.sh` script, but there's no `runtime/3rdparty/hddl` directory which is supposed to have the `install_IVAD_VPU_dependencies.sh` script ([mentioned in the guide](https://docs.openvino.ai/2022.3/openvino_docs_install_guides_installing_openvino_ivad_vpu.html#linux)) that I need to configure the HDDL plugin. 

I tried running the `install_dependencies/install_openvino_dependencies.sh` (as root user) instead, but that does not seem to work. Expected successful behavior is the Movidius MyriadX VPU is listed in `available_devices` property of object `ie = openvino.runtime.Core()`.

**Reply by ilya-lavrenov** [reply]
MYRAD and HDDL plugins are not part of 2022.3 release. Use 2022.2

**Reply by ibrahimsoliman97** [reply]
Will MYRAID & HDDL no longer be available for future versions, or this has only happened for 2022.3? @ilya-lavrenov

**Reply by ilya-lavrenov** [reply]
In 2023 and later it will not be available.
We plan to release it in 2022.3.1 LTS release and it will be available only in 2022.3.x release series.

**Reply by ibrahimsoliman97** [reply]
Just to double confirm, because we have some applications that depend on VPU/HDDL. After 2022.3.x, Openvino will stop the support for VPU/HDDL plugins, so we need to stick with 2022.3.x versions if we want to use them. am I right? and if we want to upgrade to v2023.x and above, we should change our code to use CPU or other available hardware (Not VPU). @ilya-lavrenov

**Reply by ilya-lavrenov** [reply]
right. Please, refer to _deprecation_ section https://www.intel.com/content/www/us/en/developer/articles/release-notes/openvino-relnotes.html.

Also:
`NOTE: OpenVINO 2022.3.1 LTS (coming soon) includes support for the devices listed above. Long Term Support (LTS) for OpenVINO 2022.3 ends in December 2024.  `

**Reply by ibrahimsoliman97** [reply]
Noted, Thanks!
